mu-plugins/10up-plugin/includes/core.php

```diff
@@ -22,7 +23,7 @@ function setup() {
 	};
 
 	add_action( 'init', $n( 'i18n' ) );
-	add_action( 'init', $n( 'init' ) );
+	add_action( 'init', $n( 'init' ), 8 );
```

Comment:
@darylldoyle I would add a filter hook to allow control over the `init` priority.

Reply:
Makes sense!